packages/common/src/types/events.ts

```diff
@@ -79,3 +79,13 @@ export const GetEventImageRequestParamsSchema = Type.Object({
   eventId: Type.String()
 })
 export type GetEventImageRequestParams = Static<typeof GetEventImageRequestParamsSchema>
+
+export const PostEventImageParamsSchema = Type.Object({
+  eventId: Type.String()
+})
+export type PostEventImageParams = Static<typeof PostEventImageParamsSchema>
+
+export const PostEventImageReplyBodySchema = Type.Object({
```

Comment:
The word `Body` should not be there

Reply:
![image](https://user-images.githubusercontent.com/63894614/155135839-9ddc635a-c3fb-45e1-82c9-56bc2413fe2e.png)